test/threads_test.py

```diff
@@ -25,14 +25,31 @@ def f2(x):
         self.assertEqual(fr.result, 2)
         self.assertEqual(fr2.result, 3)
 
-    def todo_test_do(self):
+    def test_do(self):
 
         # __doc__ (as of 2008-06-28) for pygame.threads.WorkerQueue.do:
 
-        # puts a function on a queue for running later.
+        # puts a function on a queue for running _later_.
         #
+        def sleepTest():
+            time.sleep(1)
 
-        self.fail()
+        def calcTest(x):
+            return x + 1
+
+        wq = WorkerQueue(num_workers=1)
```

Comment:
`wq` is too short a variable name for pylint, just change it to `worker_queue` and it'll be happy. I know half the rest of this file probably breaks PEP8 too, pygame is undergoing a gradual transition to the modern world.

Reply:
Fixed! Left the functions outside the scope of this PR untouched for now.